ext/standard/basic_functions_arginfo.h

```diff
@@ -3197,6 +3198,7 @@ static const zend_function_entry ext_functions[] = {
 	ZEND_FE(fseek, arginfo_fseek)
 	ZEND_FE(ftell, arginfo_ftell)
 	ZEND_FE(fflush, arginfo_fflush)
+	ZEND_FE(fsync, arginfo_fflush)
```

Comment:
`arginfo_fflush` is not a typo, though it is lazy in the initial PR; the function signatures are identical but it should probably be aliased at least.

Reply:
This is a generated file, you need to edit `basic_functions.stub.php` instead, which will then generate the right arginfo for the function.

Reply:
Added to stub file, is this header regenerated automatically as part of make?

Reply:
Added to stub, regenerated header